analyzers/tests/SonarAnalyzer.Test/TestCases/Utilities/MetricsAnalyzer/Razor.razor

```diff
@@ -1,8 +1,8 @@
-@page "/razor"                                     // FN: line of code, comment (location mapping issue)
-@namespace RazorNamespace                          // FN: line of code, comment (location mapping issue)
```

Comment:
Why those comments were removed?

Reply:
They were false negatives that now are fixed. See in the `MetricsAnalyzerTest.cs` the new detected lines (1 and 2).